app/jsx/components/TabAuthorComp.jsx

```diff
@@ -16,31 +17,104 @@ class CitationComp extends React.Component {
 }
 
 class TabAuthorComp extends React.Component {
+  // ToDo: Maybe componentize this as similar logic is also being used in JournalLayout
+  customIssueTitle = (vol, iss, numbering, issue_title) => {
+    if (vol == "0" && iss == "0") {
+      return issue_title 
+    }
+    else {
+      let voliss
+      if (!numbering) {
+        voliss = iss ? vol + "(" + iss + ")" : vol
+      } else if (numbering === "volume_only") {
+        voliss = "Volume " + vol 
+      } else {
```

Comment:
This took me a while to process, but I get it: "issue_only" is the only other choice, so plain 'else' works. Got it.